database/index2.js

```diff
@@ -0,0 +1,20 @@
+const { Pool } = require('pg')
+const pool = new Pool({
+    database: 'qa'
+});
+pool.connect();
+
+const getAllQuestionsWithTripId = function (query, callback) {
+    pool
+        .query(query)
+        .then((res) => {
+            callback(res.rows);
+        })
+        .catch(e => {
+            console.log(e);
```

Comment:
may want to apply callback(err) so you can send error code back to client in case of err?